roles/openshift_external_secrets/tasks/aws_backend.yaml

```diff
@@ -0,0 +1,19 @@
+- name: Create AWS keys for user account which talks to AWS SM
+  k8s:
+    definition: "{{ lookup('template', 'aws-secret-manager-users-keys.yaml') }}"
+  when: external_secrets_oidc_enabled: false
```

Comment:
Don't know if this syntax is correct. Should it be:
```
when: external_secrets_oidc_enabled
```
The change is required in multiple places.

Reply:
Yup, it makes sense!